### DISCUSSION TITLE
Service required for workspace navigation

### DISCUSSION THREAD

**Discussion by kaisalmen** [CodinGame/monaco-vscode-api#425] [Q&A] [answered]
Hi @CGNonofr do know by any chance which services are need to make workspace navigaton work (⬇️ Go Back / Go Forward) or what are the minimum requirements / required services?
This question came up here: https://github.com/eclipse-langium/langium-website/issues/227
![image](https://github.com/CodinGame/monaco-vscode-api/assets/21079138/6e65c850-db8a-49bc-ac78-b688f51a6c9a)

**Comment by CGNonofr** [comment, marked as answer]
This feature is a workbench feature, it's added by the view-common service override (used by either the workbench or the views service overrides)
